components/l10n/common/ofac_sanctioned_iso_3166_1_country_code_constants.h

```diff
@@ -23,36 +24,57 @@ inline constexpr auto kOFACSactionedISO31661CountryCodes =
 
         // ISO 3166-1 alpha-2 country codes. See
         // https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2.
+        "AL",  // Albania
+        "BA",  // Bosnia and Herzegovina
         "BY",  // Belarus
         "CU",  // Cuba
         "IR",  // Iran
         "KP",  // North Korea
         "MD",  // Moldova
+        "ME",  // Montenegro
+        "MK",  // North Macedonia
+        "RS",  // Serbia
         "RU",  // Russia
         "SY",  // Syria
         "UA",  // Ukraine (includes Crimea, Donetsk, and Luhansk regions)
+        "VE",  // Venezuela
+        "XK",  // Kosovo
 
         // ISO 3166-1 alpha-3 country codes. See
         // https://en.wikipedia.org/wiki/ISO_3166-1_alpha-3.
+        "ALB",  // Albania
+        "BIH",  // Bosnia and Herzegovina
         "BLR",  // Belarus
         "CUB",  // Cuba
         "IRN",  // Iran
+        "KOR",  // North Korea
         "MDA",  // Moldova
-        "PRK",  // North Korea
+        "MNE",  // Montenegro
+        "MKD",  // North Macedonia
+        "SRB",  // Serbia
         "RUS",  // Russia
         "SYR",  // Syria
         "UKR",  // Ukraine (includes Crimea, Donetsk, and Luhansk regions)
+        "VEN",  // Venezuela
+        "XXK",  // Kosovo
 
         // ISO 3166-1 numeric-3 country codes. See
         // https://en.wikipedia.org/wiki/ISO_3166-1_numeric.
+        "008",  // Albania
+        "070",  // Bosnia and Herzegovina
         "112",  // Belarus
         "192",  // Cuba
         "364",  // Iran
         "408",  // North Korea
         "498",  // Moldova
+        "499",  // Montenegro
+        "807",  // North Macedonia
+        "688",  // Serbia
         "643",  // Russia
         "760",  // Syria
-        "804"   // Ukraine (includes Crimea, Donetsk, and Luhansk regions)
+        "804",  // Ukraine (includes Crimea, Donetsk, and Luhansk regions)
+        "862",  // Venezuela
+        "412"   // Kosovo
```

Comment:
Same with `412` and https://en.wikipedia.org/wiki/ISO_3166-1_numeric?

Reply:
https://skosmos.loterre.fr/9SD/en/page/-J7NDWS3R-Q?clang=fr

Reply:
Kosovo is ongoing and some pages do not specify yet. Thanks

Reply:
Updated PR with additional Wiki links, thanks